posts/tests/test_models.py

```diff
@@ -71,3 +71,31 @@ def test_post_association(self):
             content_type=COMMENT_DATA['content_type'],
         )
         self.assertEqual(len(self.post.comment_set.all()), 1)
+
+
+class LikeTests(TestCase):
+    def setup(self):
+        self.user = get_user_model().objects.create_user(username=CURRENT_USER, password='password')
+        self.post = Post.objects.create(
+            title=POST_DATA['title'],
+            description=POST_DATA['description'],
+            content_type=POST_DATA['content_type'],
+            content=POST_DATA['content'],
+            author_id=get_user_model().objects.get(username=CURRENT_USER).id,
+            unlisted=POST_DATA['unlisted'])
+
+    def test_post_association(self):
+        self.user = get_user_model().objects.create_user(username=CURRENT_USER, password='password')
+        self.post = Post.objects.create(
+            title=POST_DATA['title'],
+            description=POST_DATA['description'],
+            content_type=POST_DATA['content_type'],
+            content=POST_DATA['content'],
+            author_id=get_user_model().objects.get(username=CURRENT_USER).id,
+            unlisted=POST_DATA['unlisted'])
+
+        Like.objects.create(
+            author_id=self.user.id,
+            post_id=self.post.id,
+        )
+        self.assertEqual(len(self.post.like_set.all()), 1)
```

Comment:
```suggestion
```
Please revert this one too

Reply:
wait, we don't need these tests?

Reply:
These are model tests, we already have them up above

Reply:
okk